src/style.css

```diff
@@ -0,0 +1,87 @@
+@font-face {
+  font-family: Avenir;
+  src: url("fonts/Avenir.ttc");
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  src: url("fonts/Avenir-Book.ttf");
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 900;
+  src: url("fonts/Avenir-Heavy.ttf");
+}
+
+body {
+  box-sizing: border-box;
```

Comment:
It'd be better to move it to `*` selector to make it universal for all of the elements, remove it from all the classes
```
	*{
		box-sizing: border-box;
	}
```